include/rdma/fi_prov.h

```diff
@@ -68,6 +68,7 @@ struct fi_provider {
 	int	(*fabric)(struct fi_fabric_attr *attr, struct fid_fabric **fabric,
 			void *context);
 	void	(*cleanup)(void);
+	struct fi_context context;
```

Comment:
Please move up before any function calls (before or after name), so that new calls can be added to the end without splitting them up.